index.html

```diff
@@ -77,67 +78,76 @@ <h2>Examples of Usage</h2>
+          <li>If the <dfn data-cite="ecmascript#surrounding-agent">surrounding agent</dfn>'s <dfn data-cite="html#event-loop">event loop</dfn> has any <dfn data-cite="html#task-queue">task queues</dfn> that contain a <dfn data-cite="html#concept-task">task</dfn> that will <dfn data-cite="dom#concept-event-dispatch">dispatch</dfn> an event that is a <dfn data-cite="uievents#idl-uievent">UIEvent</dfn> or <dfn data-cite="touch-events#touchevent-interface">TouchEvent</dfn> or a child of either, or if the user agent determines that it MAY soon enqueue such a task, and the dispitached event would have an EventTarget that exists within a window of the same origin, then the user agent MUST run the following steps:<ol>
+            <li> If the user agent considers the task to be a <code><a>ContinousEvent</a></code> and <i>isInputPendingOptions.includeContinuous</i> is false continue to step 6.</li>
+            <li>The user agent MAY let <i>pendingResult</i> be true.</li>
           </ol></li>
-          <li>Return <code>pendingResult</code>.</li>
+          <li>Return <i>pendingResult</i>.</li>
         </ol>
-        <p class="note">The overall goal for this api is to allow user agents to return true if they think an event may end up getting dispatched to a browsing context soon. Not only if an event is already placed into the queue. This may mean that a user agent may look into the current configuration of iframes, other events in the queue, or other information when making decisions in step 5b. For example, there may be cases where if a mousedown event is queued then a click event may be fired for this frame after the mousedown event. However after the mousedown event is fired the user agent knows that the click event will not be fired. In this case the user agent should return 'true' for a 'click' input before the mousedown event, but after it knows that click will not be fired it should return false.</p>
+        <p class="note">The overall goal for this api is to allow user agents to return true if they think an event may end up getting dispatched to a browsing context soon. Not only if an event is already placed into the queue. This may mean that a user agent may look into the current configuration of iframes, other events in the queue, or other information when making decisions in step 4.1. For example, there may be cases where if a mousedown event is dispatched then a click event may be fired for this frame after the mousedown event. However if the mousedown event is canceled then the user agent knows that the click event will not be fired. In this case the user agent should return 'true' for a 'click' input before the cancling mousedown event, but after it knows that a click will not be fired it should return false.</p>
```

Comment:
```suggestion
        <p class="note">The overall goal for this api is to allow user agents to return true if they think an event may end up getting dispatched to a browsing context soon, and not only if an event is already placed into the queue. This may mean that a user agent may look into the current configuration of iframes, other events in the queue, or other information when making decisions in step 4.1. For example, there may be cases where if a mousedown event is dispatched then a click event may be fired for this frame after the mousedown event. However if the mousedown event is canceled then the user agent knows that the click event will not be fired. In this case the user agent should return 'true' for a 'click' input before the canceling mousedown event, but after it knows that a click will not be fired it should return false.</p>
```